### ISSUE TITLE
[Feature Request] Scrollbar arrows are missing. 

### ISSUE THREAD

**Issue by simsek8** [telegramdesktop/tdesktop#8080]
### Is your feature request related to a problem?
There is no scrollbar arrows. This is a problem for scrolling precisely when your scroll wheel is broken which is a common problem.
![telegram](https://user-images.githubusercontent.com/26023956/84772406-8f9ba280-afe3-11ea-95f6-4faaacddf757.png)

### Describe the solution you'd like
There could be tiny arrows for scrolling.

### Additional context
Using Telegram 2.1.10 on Windows 10 Home.

**Comment by Aokromes** [comment]
as workaround you can click and drag without mouse whell for scrolling.

**Comment by simsek8** [comment]
> as workaround you can click and drag without mouse whell for scrolling.

But it is not precise as arrow clicking.

**Comment by simsek8** [comment]
> Hey there!
> 
> This issue will be automatically closed in 7 days if there would be no activity. We therefore assume that the user has lost interest or resolved the problem on their own.
> 
> Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the issue.
> 
> Thanks!

No. The problem still exist. It is so dumb not to have arrows in huge message load.

**Comment by ubone** [comment]
Please add those arrows (to page up/down). This is a desktop application. Not a phone one where you scroll with your thumb.

Optionally you could do what GTK does: scroll one page if clicked with the right mouse button.
